article/Method/alpha-k-layer-combination.tex

```diff
@@ -81,6 +49,12 @@ \subsubsection{Degree based layer combination}
     \label{alg:aggresive-layereffect}
 \end{algorithm}
 
+\paragraph{Balanced Layer Combination}
+The pseudo code for the balanced layer combination (BLC) algorithm can be seen on \autoref{alg:bal-layereffect}.
+The BLC algorithm works much the same as ALC but instead of popping a given layer of the list when its effect has been reduced it is instead kept.
```

Comment:
```suggestion
The BLC algorithm works much the same as ALC but instead of popping a given layer of the list when its effect has been reduced, it is instead kept.
```